pytorch3d/renderer/points/pulsar/unified.py

```diff
@@ -266,7 +270,10 @@ def _extract_intrinsics(  # noqa: C901
                 afov = kwargs.get("fov", cameras.fov)[cloud_idx]
                 if kwargs.get("degrees", cameras.degrees):
                     afov *= math.pi / 180.0
```

Comment:
@classner This operation shouldn't be in-place I think!
Replacing `afov *= math.pi / 180.0` with `afov = afov * math.pi / 180.0` fixed issues with my code.
Thanks for this PR by the way!